src/test/components/TimelineMarkers.test.js

```diff
@@ -43,7 +43,7 @@ it('renders TimelineMarkers correctly', () => {
   const profile = getProfileWithMarkers([
     ['Marker A', 0, { startTime: 0, endTime: 10 }],
     ['Marker B', 0, { startTime: 0, endTime: 10 }],
-    ['Marker C', 0, { startTime: 5, endTime: 15 }],
+    ['Marker C', 5, { startTime: 5, endTime: 15 }],
```

Comment:
I don't think we use the `time` property, but anyway it should be the same as startTime for consistency.